### DISCUSSION TITLE
mount error after install blueCFD

### DISCUSSION THREAD

**Discussion by yebin-nam** [blueCFD/Core#249] [Q&A]
when i start blueCFD -terminal then I got bottom message
"
mount: can't find /home/ofuser/blueCFD in /etc/fstab or in /etc/fstab.d/$USER
mount: /home/ofuser/AddOns: Invalid argument
---------- Setting up OpenFOAM variables for new build ----------
-- Current MPI chosen for parallel operations: MSMPI71
Choose one option for selecting the new build architecture:
    1 - OpenFOAM 5.x mingw-w64 Double Precision (of5-64)
    2 - OpenFOAM 5.x mingw-w64 Single Precision (of5-64S)
    3 - OpenFOAM 5.x mingw-w32 Double Precision (of5-32)
    4 - OpenFOAM 5.x mingw-w32 Single Precision (of5-32S)
Choose one of the numbers 1 to 4 (any other for none) then press Enter:
"
after i press 1, 
"
bash: /home/ofuser/blueCFD/OpenFOAM-5.x/etc/bashrc: No such file or directory
Environment is now ready. Notes:
   - You can change between installed versions by running: ofmenu
   - You can change to other predefined versions by running: ofmenuNew
"
I want to start blueCFD... plz help me!

**Comment by wyldckat** [comment]
I need more details:
1. Which Windows version are you using?
2. In which language is your Windows installation running in?
3. Did you install as normal user or with administrative permissions?
4. Where is blueCFD-Core installed?
    * I need the complete path.
    * You can find where is was installed, if you go to the Windows Start Menu, go into the folder `blueCFD-Core 2017` and click on the `blueCFD-Core folder` shortcut listed there. It should open the Windows File Explorer in the correct installation folder for `blueCFD-Core 2017`, for example: `C:\Program Files\blueCFD-Core-2017`

**Comment by WangShekun** [comment]
I had the same problem, I installed in the “Program files” folder, but the Bluecfd installation requires that the installation folder should not have spaces, so I changed the folder, and that solved the problem, I don't know if you have the same problem as me, I hope this helps.
